tools/rdm/TestMixins.py

```diff
@@ -907,3 +976,49 @@ def VerifyResult(self, response, fields):
            self.DESCRIPTION_FIELD,
            self.current_item,
            fields[self.DESCRIPTION_FIELD].encode('string-escape')))
+
+
+class GetSettingDescriptionsMixinRange(GetSettingDescriptionsMixin):
```

Comment:
The convention I've tried to maintain is that mixins end with 'Mixin', so use GetSettingDescriptionsRangeMixin.

Same for the one below as well